guides/common/attributes-base.adoc

```diff
@@ -36,6 +36,7 @@
 :ansible-galaxy: https://galaxy.ansible.com/theforeman/foreman
 :ansible-namespace-example: theforeman.foreman._module_name_
 :ansible-namespace: theforeman.foreman
+:ansible-galaxy-name: Ansible Galaxy
```

Comment:
Can you move this below `:ansible-galaxy:` to order it the same way way as in `attributes-satellite`?